Backend/dev/migrations-read-only/dynamic-offer-driver-app/transporter_config.sql

```diff
@@ -338,4 +338,13 @@ ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD COLUMN recompute_dista
 
 ------- SQL updates -------
 
-ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD COLUMN min_threshold_for_pass_through_destination int ;
\ No newline at end of file
+ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD COLUMN min_threshold_for_pass_through_destination int ;
+
+
+------- SQL updates -------
+
+ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD COLUMN fleet_alert_threshold integer ;
```

Comment:
This can be moved to FleetConfig, it's created in @00abhishek0000 PR, will be merging it today